nginx-log-metrics/README.md

```diff
@@ -0,0 +1,66 @@
+# Nginx log metrics demo
+
+## Quick Start
+
+Build and start with proxy:
+
+```shell
+docker compose build --build-arg https_proxy=http://172.17.0.1:7890
+docker compose up -d --force-recreate
+```
```

Comment:
We will need to change this part into a more user-facing section like `How it works` focusing more on introduction of our features. We can do it after this being merged.